specification/migrate/resource-manager/Microsoft.OffAzure/stable/2020-07-07/migrate.json

```diff
@@ -2968,16 +3450,226 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/OperationResultList"
+              "$ref": "#/definitions/PrivateLinkResourceCollection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Private link resources per site": {
+            "$ref": "./examples/PrivateLinkResources_ListBySite.json"
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.OffAzure/VMwareSites": {
+      "get": {
+        "summary": "Get all vmware sites.",
+        "description": "Get all the vmware sites in the subscription.",
+        "operationId": "VMwareSites_ListBySubscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "description": "The API version to use for this operation.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "headers": {
+              "x-ms-request-id": {
+                "type": "string",
+                "description": "Service generated Request ID."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/VMWareSitesResultList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Projects_List": {
+            "$ref": "./examples/VMwareSitesInSubscription_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.OffAzure/VMwareSites": {
```

Comment:
>VMwareSites [](http://example.com/codeflow?start=101&length=11)

You should also have a GET operation for a single resource of type `Microsoft.OffAzure/VMwareSites`

Reply:
Its there, "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OffAzure/VMwareSites/{siteName}", line 1836

Reply:
I missed it before. Thanks for pointing it out!